app/src/main/res/layout/activity_login.xml

```diff
@@ -40,6 +40,7 @@
         <android.support.v7.widget.AppCompatEditText
             android:layout_width="match_parent"
             android:layout_height="49dp"
+            android:id="@+id/user_editText"
```

Comment:
Can you name this "etUsername" instead and for the password "etPassword" to follow the convention used in the rest of the project?

Reply:
Hi @isabelcosta I have made all the changes requested by you :100: Have a look